crates/pipeline-manager/src/config.rs

```diff
@@ -169,6 +177,16 @@ pub struct CommonConfig {
     /// Port used by the runner to both bind its HTTP server, and on which it can be reached.
     #[arg(long, default_value_t = 8089)]
     pub runner_port: u16,
+
+    /// How many HTTP worker threads to spawn for each HTTP runtime system.
+    ///
+    /// If not specified, it will default to using the number of available CPUs
+    /// in the system.
+    ///
+    /// Note that the pipeline-manager may start up to three HTTP servers
+    /// which will lead to a total of `3 * http_workers` threads.
+    #[arg(long, default_value_t = default_http_workers(), env = "FELDERA_HTTP_WORKERS")]
+    pub http_workers: usize,
```

Comment:
I imagined that pipeline-manager main would just be a single-threaded async runtime, and then each of the three mains would start their own runtime independently without inheriting it, to isolate the mains from one another.

Reply:
is that how it works today? I couldn't tell from the code